pygotham/talks/models.py

```diff
@@ -98,3 +94,20 @@ def __str__(self):
     def is_accepted(self):
         """Return whether the instance is accepted."""
         return self.status == 'accepted'
+
+
+class CoPresenter(db.Model):
+
+    __tablename__ = 'copresenters'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    name = db.Column(db.String(255), info={'label': 'Name'}, nullable=False)
```

Comment:
I thought the decision was made so that a co-presenter did not have to be a user? A user would just email us and we would either create a co-presenter with their name or link their account through this table if possible?